prisma/prisma.js

```diff
@@ -0,0 +1,14 @@
+import { PrismaClient } from '@prisma/client';
+
+let prisma;
+
+if (process.env.NODE_ENV === 'production') {
+  prisma = new PrismaClient();
+} else {
+  if (!global.prisma) {
```

Comment:
where is global coming from? just curious

Reply:
okay cool. thanks for clarification

Reply:
Thanks for adding date object

Reply:
yeah, it's very important. Good catch